### PR TITLE
Make the entire realization box go green on finish

### LINKED ISSUE #6190: ERT gui does not fill box when all jobs are done
**Describe the bug**
When a realization completes all jobs, it is supposed to color everything green. However, this does not happen when the amount of boxes don't add up. For example, if we have three jobs then the box will be 2x2 cells. When all jobs are completed, only 3/4 of the box will be filled. (see attached image). The extra cells needed to fill the box remain gray. 

**To reproduce**
Steps to reproduce the behaviour:
1. run ert gui with a non-square number of jobs

**Expected behaviour**
When all jobs are completed, the entire box should be colored green if successful or red if failed.
**Screenshots**
image

**Environment**
 - OS: [e.g. RHEL7]
 - ERT/Komodo release: [e.g. 2.18 or 2020.12.00]
 - Python version
 - Remote/HPC execution involved: [yes|no]

**Additional context**

### PR DESCRIPTION
**Pull request by larsevj**
**Issue**
Resolves #6190 

## Pre review checklist

- [x] Read through the code changes carefully after finishing work
- [x] Make sure tests pass locally (after every commit!)
- [x] Prepare changes in small commits for more convenient review (optional)
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Updated documentation
- [x] Ensured that unit tests are added for all new behavior (See 
    [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)),
    and changes to existing code have good test coverage.

## Pre merge checklist
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by larsevj** [comment]
GUI behaviour on failed job:
image
GUI behaviour on finished realizations:
image

File: src/ert/gui/simulation/view/realization.py [hunk 1]
```diff
@@ -144,6 +153,9 @@ def _paint_inner_grid(self, painter: QPainter, rect: QRect, colors) -> None:
         else:
             foreground_image = _image_cache[colors_hash]
 
+        border_pen.setWidth(2)
+        painter.setPen(border_pen)
+        painter.drawRect(rect)
```
Comment (andreas-el) [inline src/ert/gui/simulation/view/realization.py:158]: Why pass the pen? We could avoid doing so?
```
pen = QPen(QColorConstants.Grey)
pen.setWidth(2)
painter.setPen(pen)
painter.drawRect(rect)
```

Also I realised that the default pen is black and width (1).
So we could replace this;

```
        border_pen = QPen()
        border_pen.setColor(QColorConstants.Black)
        border_pen.setWidth(1)
```

**Review by andreas-el — CHANGES_REQUESTED** [review]
Looks good. Could consider removing passing the pen.

File: src/ert/gui/simulation/view/realization.py (hunk 1 above)
Comment (larsevj) [reply src/ert/gui/simulation/view/realization.py:158]: Are you sure about the default pen? Removing the border_pen results in a white border for me.

Comment (larsevj) [reply src/ert/gui/simulation/view/realization.py]: I removed the pen from the function arguments. I also initialized the pens in-place, maybe it is a bit harder to read?

**Review by andreas-el — APPROVED** [review]
Good job! 🚀

**Comment by DanSava** [comment]
Don't forget to squash the commits before merging

**Pull request merged**